package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "jStorage",
```

Comment:
This is one npm package for the whole repo right?
We discussed on having  jStorage.github as one package that was dependent on jStorage.

Why not do it like that?